src/Arriba/Arriba.Server/Extensions/OAuthExtension.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Arriba.Configuration;
+using Arriba.Security.OAuth;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Arriba.Server.Extensions
```

Comment:
Please update the default namespace for the project. The base namespace should be Arriba not Arriba.Server. This way this file falls into Arriba.Extensions instead of Arriba.Server.Extensions. You don't need to correct all other changes, but we should do that in another change soon.